src/types.jl

```diff
@@ -149,12 +507,552 @@ for Model in [:NeuralNetworkRegressor, :MultitargetNeuralNetworkRegressor]
+
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `training_losses`: The history of training losses, a vector containing the history of all the losses during training. The first element of the vector is the initial penalized loss. After the first element, the nth element corresponds to the loss of epoch n-1.
+  all the losses during training. The first element of the vector is the initial penalized loss. After the first element, the nth element corresponds to the loss of epoch n-1.
+  penalized loss. After the first element, the nth element corresponds to the loss of epoch n-1.
+  epoch n-1.
+# Examples
+
+In this example we build a regression model using the Boston house price dataset
+```julia
+  using MLJ
+  using MLJFlux
+  using Flux
+  using Plots
+```
+First, we load in the data, with target `:MEDV`. We load in all features except `:CHAS`:
+```julia
+data = OpenML.load(531); # Loads from https://www.openml.org/d/531
+
+y, X = unpack(data, ==(:MEDV), !=(:CHAS); rng=123);
+
+scitype(y)
+schema(X)
+```
+Since MLJFlux models do not handle ordered factors, we can treat `:RAD` as `Continuous`:
+```julia
+X = coerce(X, :RAD=>Continuous)
+```
+Lets also make a test set:
+```julia
+(X, Xtest), (y, ytest) = partition((X, y), 0.7, multi=true);
+```
+Next, we can define a `builder`. In the following macro call, `n_in` is the number of expected input features, and rng is a RNG. `init` is the function used to generate the random initial weights of the network.
+expected input features, and rng is a RNG. `init` is the function used to generate the random initial weights of the network.
+random initial weights of the network.
+```julia
+builder = MLJFlux.@builder begin
+  init=Flux.glorot_uniform(rng)
+  Chain(Dense(n_in, 64, relu, init=init),
+        Dense(64, 32, relu, init=init),
+        Dense(32, 1, init=init))
+end
+```
+Finally, we can define the model!
+```julia
+NeuralNetworkRegressor = @load NeuralNetworkRegressor
+  model = NeuralNetworkRegressor(builder=builder,
+                                 rng=123,
+                                 epochs=20)
+```
+For our neural network, since different features likely have different scales, if we do not standardize the network may be implicitly biased towards features with higher magnitudes, or may have [saturated neurons](https://www.informit.com/articles/article.aspx%3fp=3131594&seqNum=2)  and not train well. Therefore, standardization is key!
+not standardize the network may be implicitly biased towards features with higher magnitudes, or may have [saturated neurons](https://www.informit.com/articles/article.aspx%3fp=3131594&seqNum=2)  and not train well. Therefore, standardization is key!
+magnitudes, or may have [saturated neurons](https://www.informit.com/articles/article.aspx%3fp=3131594&seqNum=2)  and not train well. Therefore, standardization is key!
+neurons](https://www.informit.com/articles/article.aspx%3fp=3131594&seqNum=2)  and not train well. Therefore, standardization is key!
```

Comment:
I think this is too much detail. We're not teaching ML principles here, just demonstrating how to use the software. 

How about:

```suggestion
We will arrange for standardizaion of the the target by wrapping our model
 in `TransformedTargetModel`, and standardization of the features by 
inserting the wrapped model in a pipeline:
```